src/openrct2/CmdlineSprite.cpp

```diff
@@ -88,6 +78,23 @@ void SpriteFile::MakeEntriesRelative()
     isAbsolute = false;
 }
 
+void SpriteFile::AddImage(ImageImporter::ImportResult& image)
+{
+    Header.num_entries++;
+    Header.total_size += static_cast<uint32_t>(image.Buffer.size());
+    Entries.reserve(Header.num_entries);
+
+    MakeEntriesRelative();
+    Data.reserve(Header.total_size);
+    MakeEntriesAbsolute();
```

Comment:
(On commit https://github.com/OpenRCT2/OpenRCT2/pull/13590/commits/8a91330eb7df1e56de3ceddce95937d94396ce82)

Another case for the scoped relative/absolute RAII class.